From d61123c2612f20ea42ad2243a59bcfe97054eb0e Mon Sep 17 00:00:00 2001
From: Nadav Hugim <nhugim@habana.ai>
Date: Tue, 05 Apr 2022 16:16:07 +0300
Subject: [PATCH] [SW-69376] erom: change erom bin to little endian

the erom fit burn swap bytes so we need big endian file
that convert to little endian on the flash

Change-Id: I2c2c338081dd6d8813afb25dbe80908732ffb571
Signed-off-by: Nadav Hugim <nhugim@habana.ai>
---

diff --git a/board/habanalabs/gaudi2/agent-fw_loader_erom.its b/board/habanalabs/gaudi2/agent-fw_loader_erom.its
index fbf7f64..db31304 100644
--- a/board/habanalabs/gaudi2/agent-fw_loader_erom.its
+++ b/board/habanalabs/gaudi2/agent-fw_loader_erom.its
@@ -14,7 +14,7 @@
 	images {
 		erom {
 			description = "Erom";
-			data = /incbin/("@HL_DEV@-erom.bin.le");
+			data = /incbin/("@HL_DEV@-erom.bin.be");
 			type = "standalone";
 			compression = "none";
 			offset = <0>;
